example/odoo/Dockerfile

```diff
@@ -16,6 +16,7 @@ COPY ./setup.py /odoo/
 COPY ./VERSION /odoo/
 COPY ./migration.yml /odoo/
 
+RUN replace_dependencies.sh
```

Comment:
Shouldn't we put it in the entrypoint?

Reply:
Too late, it has to be done before the next 2 lines in this file

Reply:
Right!